presto-main/src/main/java/com/facebook/presto/operator/FileFragmentResultCacheManager.java

```diff
@@ -281,13 +294,35 @@ public int hashCode()
         }
     }
 
+    private static class CacheValue
```

Comment:
Maybe we could call this `FileInfo`? Both `path` and `size` is part of file information :)

Reply:
fix